commands/get/project.go

```diff
@@ -0,0 +1,111 @@
+package get
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"text/template"
+
+	"github.com/feedhenry/rhm/commands"
+	"github.com/feedhenry/rhm/storage"
+	"github.com/urfave/cli"
+)
+
+var projectTemplate = `
```

Comment:
Yeah I was thinking this too 👍 
https://github.com/feedhenry/rhm/issues/17
Feel free to modify it as it on mentions json